### ISSUE TITLE
Can't resolve `fs`

### ISSUE THREAD

**Issue by rblalock** [danomatic/react-pdf-html#11]
```
./node_modules/css/lib/stringify/source-map-support.js:9:0
Module not found: Can't resolve 'fs'
```

Does this require server side rendering?

**Comment by danomatic** [comment]
@rblalock I haven't tried it anywhere but in an Express app. I don't know why `css` is trying to load `fs`, but I just released `react-pdf-html` version `1.1.1` that might fix your problem. Please try it and let me know.

**Comment by rblalock** [comment]
Howdy.  Just tried and same problem.  It is curious that it's trying to use `fs`.  I'll dive through some of the code when I get a chance and try to understand why that library would think it should use fs

**Comment by danomatic** [comment]
It’s related to some part of the css library that can load source maps. I think we might need to switch to another css parser

**Comment by danomatic** [comment]
This should be fixed with the switch to `css-tree` in `v1.1.2`
